### ISSUE TITLE
tools install failing on latest kali

### ISSUE THREAD

**Issue by analyserdmz** [epi052/recon-pipeline#105]
**Describe the bug**
Tools installation fails on latest kali linux. When Go fails, none of the tools are installed as well. Trying by following the readme file.

**To Reproduce**
1. Follow the readme instructions

**Expected behavior**
Tools should be installed with no problems from withing the recon-pipeline prompt.

**Traceback / Error Output**
```
recon-pipeline> tools install all
[-] go queued
[-] webanalyze queued
[-] exploitdb queued
[-] searchsploit queued
[-] recursive-gobuster queued
[-] aquatone queued
[-] tko-subs queued
[-] seclists queued
[-] waybackurls queued
[-] masscan queued
[-] subjack queued
[-] gobuster queued
[-] luigi-service queued
[-] amass queued
[*] Installing go...
[=] wget -q https://dl.google.com/go/Could not find latest go version download url -O /tmp/go.tar.gz
[=] tar -C /root/.local/recon-pipeline/tools/pipeline-go -xvf /tmp/go.tar.gz
[!] tar: This does not look like a tar archive

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
[=] bash -c 'if [ ! $(grep $(dirname /root/.local/recon-pipeline/tools/pipeline-go/go/bin/go ) /root/.bashrc ) ]; then echo PATH=${PATH}:$(dirname /root/.local/recon-pipeline/tools/pipeline-go/go/bin/go ) >> /root/.bashrc ; fi'
[!!] one (or more) of go's commands failed and may have not installed properly; check output from the offending command above...
[!] webanalyze has an unmet dependency; installing go
[*] Installing go...
[=] wget -q https://dl.google.com/go/Could not find latest go version download url -O /tmp/go.tar.gz
[=] tar -C /root/.local/recon-pipeline/tools/pipeline-go -xvf /tmp/go.tar.gz
[!] tar: This does not look like a tar archive

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
[=] bash -c 'if [ ! $(grep $(dirname /root/.local/recon-pipeline/tools/pipeline-go/go/bin/go ) /root/.bashrc ) ]; then echo PATH=${PATH}:$(dirname /root/.local/recon-pipeline/tools/pipeline-go/go/bin/go ) >> /root/.bashrc ; fi'
[!!] one (or more) of go's commands failed and may have not installed properly; check output from the offending command above...
[*] Installing webanalyze...
[=] /root/.local/recon-pipeline/tools/pipeline-go/go/bin/go get github.com/rverton/webanalyze/...
EXCEPTION of type 'FileNotFoundError' occurred with message: '[Errno 2] No such file or directory: '/root/.local/recon-pipeline/tools/pipeline-go/go/bin/go''
To enable full traceback, run the following command: 'set debug true'
```

**Environment (please complete the following information):**
 - recon-pipeline version: v1.0.1
 - python version 3.10.5
 - OS:
```
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2022.3"
VERSION_ID="2022.3"
VERSION_CODENAME="kali-rolling"
ID_LIKE=debian
ANSI_COLOR="1;31"
HOME_URL="https://www.kali.org/"
SUPPORT_URL="https://forums.kali.org/"
BUG_REPORT_URL="https://bugs.kali.org/"
```

**Comment by spothound** [comment]
Hello @analyserdmz, in case you are still interested in this, a quick fix is proposed in this PR: https://github.com/epi052/recon-pipeline/pull/107

**Comment by epi052** [comment]
this should be fixed now, having merged #107; feel free to reopen if that's not the case
